src/vsr/replica.zig

```diff
@@ -4550,7 +4550,7 @@ pub fn ReplicaType(
                 assert(replica < self.replica_count);
             }
 
-            counter.setIntersection(quorum_counter_null);
```

Comment:
Just curious—was there a bug here, in using `setIntersection()`?

Reply:
No, there wasn't a bug there. I noticed that since these bitsets are values it is clearer to just assign `quorum_counter_null`.